sdcm/nemesis.py

```diff
@@ -3517,11 +3517,17 @@ def decommission_node(self, node):
 
     def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[bool, DefaultValue]] = DefaultValue,
                            dc_idx: Optional[int] = None):
-        for _ in range(add_nodes_number):
+        for idx in range(add_nodes_number):
             if self._is_it_on_kubernetes():
+                if rack is None and self._is_it_on_kubernetes():
+                    # currently despite setting multi az on k8s, we still use only one rack
```

Comment:
Remove this comment, as I wrote below, K8S `racks` are not tied anyhow to real platform AZs.
You can leave `rack = 0` here, but really correct approach would be to read the `scyllacluster` K8S object spec and get the num of configured `K8S racks` out of it.